.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+    - id: black
+      language_version: python3.8
```

Comment:
Black check. Sure.... if you want. In my opinion this just adds the extra step of having to type `black .` so I'd rather have it done automatically.
Also... we can set the default language for all checks to be python 3.8

Reply:
OK, it's fine to leave it as is. Now that we've sortof figured out the vscode integration I'm going to assume any changes it would make have already happened during development. I'll probably run the extra manual step anyway before committing.